### ISSUE TITLE
[Bug] Leap not working

### ISSUE THREAD

**Issue by ThatOneTraitor** [Mithion/ArsMagica2#1397]
When i cast leap, If i add 3 buff powers (to make it leap 3) it stops giving a jump boost and just shows a potion effect that does nothing.

**Comment by c62624** [comment]
Buff modifier is broken. It doesn't seem to have any effect.

#1174

**Comment by Jeffrey2911** [comment]
when i use buff power on haste it seems to work fine but when i add leap into the mix it doesnt give the effect

**Comment by ThatOneTraitor** [comment]
Yeah, but adding buff power completely nullifies leap. if you use haste and buff power x3, it will just give haste 1, but with leap, it will just not do anything
